apache/superset
# fix(logging): downgrade csv export log to debug

### SUMMARY

This should be on debug level as it logs the entire export request which may contain sensitive info in the query. 

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

N/A

### TESTING INSTRUCTIONS

- Export a CSV, this line shouldn't log.
- Set log level to DEBUG, export CSV, this line should log.

### ADDITIONAL INFORMATION
